virttest/virsh.py

```diff
@@ -1580,7 +1580,8 @@ def undefine(name, options=None, **dargs):
     if options is not None:
         cmd += " %s" % options
 
-    if platform.machine() == "aarch64":
+
+    if platform.machine() == "aarch64" or "x86_64":
```

Comment:
Please help some descriptive message for --nvram purpose and make a statement , similar like below:  it will be applied on both seabios guest (a risk, suppose no side effect currently) and uefi guest...

Reply:
updated